components/ChoreWeight.tsx

```diff
@@ -13,29 +14,37 @@ export default function ChoreWeight({
 }: ChoreWeightProps) {
   const [isPressed, setIsPressed] = useState(false);
   const [value, setValue] = useState(initialWeight);
-  const [backgroundColor, setBackgroundColor] = useState('rgba(0, 0, 0, 0.2)');
+  const { colorMode, setColorMode } = useContext(ThemeContext) as {
+    colorMode: string;
+    setColorMode: (value: string) => void;
+  };
```

Comment:
You shouldn't be using the context here, you can instead use the useTheme hook.
```suggestion
  const { colors, dark } = useTheme();
```
Something like that, have never used the "dark" part myself, but I believe you could use that to check if it's dark mode.